### ISSUE TITLE
MACC of mobilenet v2 is different from paper 

### ISSUE THREAD

**Issue by CuriousCat-7** [Lyken17/pytorch-OpCounter#66]
Hello, thank you for sharing your code. Recently I'm trying to design new architectures based on Mobilenet v2, but I found the Macc calculated by your project is different from it in the paper.

In the [paper](https://arxiv.org/pdf/1801.04381.pdf):
![image](https://user-images.githubusercontent.com/17199271/75547949-06c61e80-5a67-11ea-92cd-c12d984e3797.png)

It should be 585 MMac  and 6.9M paremeters for MobilNet V2(1.4x).
However, 
I get :
- Computational complexity:       603.134720.0 MMac
- Number of parameters:           6.108776.0 M

By the way, I get the different result by [sovrasov's code](https://github.com/sovrasov/flops-counter.pytorch)
- Computational complexity:       610.1 MMac
- Number of parameters:           6.11 M

code I used:
```python
     from torchvision.models import mobilenet_v2
     model = mobilenet_v2(width_mult=1.4)
     from thop import profile
     import torch
     input = torch.randn(1, 3, 224, 224)
     macs, params = profile(model, inputs=(input, ))
     print("macc", macs)
     print("params", params)
```
Could you tell me the reason for the difference? Thanks.

**Comment by Lyken17** [comment]
I think the main difference here, is that TensorFlow does NOT count "batchnorm" into FLOPs. If you profile with custom rules to set bn as zero, you should be able to get similar numbers.

**Comment by CuriousCat-7** [comment]
solved by 
https://github.com/sovrasov/flops-counter.pytorch/issues/33
